app/webserver/webserver.go

```diff
@@ -84,7 +84,7 @@ func syncClients() {
 		// SetGui()
 		UpdateChatList()
 		UpdateContactList()
-		UpdateActiveChat()
+		// UpdateActiveChat()
```

Comment:
If you need to keep some code commented, please explain why in the code ;)

Reply:
I wasn't shure if it works, but it works.